spot_rl_experiments/spot_rl/skills/atomic_skills.py

```diff
@@ -33,49 +38,138 @@
 # Import core classes
 from spot_wrapper.spot import Spot
 
+###
+### Atomic Skills is a skill which requires its own policy and environment and does not depend on other atomic or composite skills
+###
+
+
+"""
```

Comment:
This entire doc-string is awesome. Does this exist in a readme too that we can point to from another page? It'd be great to increase visibility of this context.

Reply:
I have moved this to a separate README.md file within skills folder.